### PR TITLE
Vanish Update

### PR DESCRIPTION
**Pull request by Aiidoneus**
Added some useful information to the vanish module. I am sure I will add
some more at a later date. Also want to see if this is the sort of thing
you are looking for. 

Let me know what you think about this for at least a start.

### CODE REVIEW

File: docs/modules/vanish.md [hunk 1]
```diff
@@ -8,4 +8,16 @@ modulename: Vanish
 
 ## Introduction
 
-The Vanish module allows players to turn themselves invisible to other entities on the server.
\ No newline at end of file
+The Vanish module allows players to turn themselves invisible to other entities on the server. This is a good way to hide yourself while doing work on your server in peace when other players are on your server. 
+
+## Configuration
+
+Don't be deceived when looking at the configuration options in your `main.conf`file. While only a couple options reside in the vanish section of your configuration file, other modules have options for `/vanish`. These modules are:
```
Comment (dualspiral) [inline docs/modules/vanish.md]: This is a very good point, but you should also mention the configuration options available for the `vanish` module.

File: docs/modules/vanish.md [hunk 2]
```diff
@@ -8,4 +8,16 @@ modulename: Vanish
...
+
+* [AFK Module](afk.html)
+* [Server-List Module](serverlist.html)
+
+## Vanish Status on Logout
+
+While Nucleus doesn't have a way to forcibly log players in as vanished while joining, it does remember what your vanish status was when you logged out. To enable this function you need to assign the `nucleus.vanish.persist` permission.
```
Comment (dualspiral) [inline docs/modules/vanish.md]: To make it clear, that last sentence should say it should be assigned to the user or group in question, I think.

**Comment by dualspiral** [comment]
In general, this is the sort of thing I'm looking for. It should be enough to tell users what it does, how to configure it, and anything that they might look for that isn't obvious at first sight.

**Comment by Aiidoneus** [comment]
To make sure I am getting this right, would you like a more in-depth explanation of what each configuration does in the `main.cfg` file? Let me work on that last sentence. =)

**Comment by dualspiral** [comment]
It doesn't need to be stupidly in depth - enough to describe the effect and possibly problems with a config option.

**Comment by Aiidoneus** [comment]
Cleaned up a bit on the vanish page with that commit. ^

**Comment by Aiidoneus** [comment]
I believe I am done with this module. If you want to merge it be my guest. I would like to have the Edit Github revision/addition on a seperate pull.

**Comment by dualspiral** [comment]
Thanks!

**Pull request merged**